tests/units/online/test_hand.py

```diff
@@ -57,3 +57,15 @@ def test_gripper_goal_position(reachy_sdk_zeroed: ReachySDK) -> None:
 
     assert np.isclose(reachy_sdk_zeroed.r_arm.gripper.goal_position, 40, 1e-01)
     assert np.isclose(reachy_sdk_zeroed.l_arm.gripper.goal_position, 70, 1e-01)
+
+
+@pytest.mark.online
+def test_gripper_off(reachy_sdk_zeroed: ReachySDK) -> None:
+    reachy_sdk_zeroed.set_pose("elbow_90")
+    reachy_sdk_zeroed.l_arm.gripper.turn_off()
+    reachy_sdk_zeroed.l_arm.goto_joints([15, 15, 0, -60, 0, 0, 0])
+    time.sleep(3.0)
+    assert reachy_sdk_zeroed.l_arm.is_on() == True
+    assert np.isclose(reachy_sdk_zeroed.l_arm.elbow.pitch.present_position, -60, 10)
+    reachy_sdk_zeroed.turn_on()
+    reachy_sdk_zeroed.set_pose("default")
```

Comment:
There is no assert after these two lines (i.e. no test). Did you forget a test?
Otherwise you can remove them